jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -185,6 +190,16 @@ private static void completeInitialization() {
 			}
 		}
 /*[ENDIF] CRAC_SUPPORT */
+		/*[IF (JAVA_SPEC_VERSION == 11) | (JAVA_SPEC_VERSION == 17)]*/
```

Comment:
Please use the "between" idiom:
```java
		/*[IF (11 <= JAVA_SPEC_VERSION) | (JAVA_SPEC_VERSION <= 17)]*/
```
Please also just use the fully-qualified name `jdk.internal.jimage.ImageReaderFactory` instead of an import for this one use.